src/components/Tabs/__tests__/Tabs.spec.js

```diff
@@ -16,6 +18,14 @@ describe('<Tabs> that renders tab container with some tabs', () => {
             </Tabs>
         );
         expect(toJson(wrapper)).toMatchSnapshot();
+
+        wrapper
+            .find('TabMenu')
+            .find('TabItem')
+            .at(1)
+            .simulate('click');
+
+        expect(onChange).not.toHaveBeenCalled();
```

Comment:
I'm not sure there is a point in this test. you create a variable you never use and then ask if anything happened to it?

Reply:
Yes, now that I'm looking at it again it doesn't seem to make much sense indeed. I'll see if I can think of something else (purpose was to check that `handleTabChange` would not trigger `onChange` even when the callback prop is not defined).

Reply:
check that the click doesn't throw... but move it to a separate test, not part of initial rendering